lib/new_relixir.ex

```diff
@@ -13,26 +13,12 @@ defmodule NewRelixir do
     import Supervisor.Spec, warn: false
 
     children = [
-      worker(:statman_server, [1000]),
-      worker(:statman_aggregator, []),
+      worker(NewRelixir.Collector, []),
+      worker(NewRelixir.Poller, [&NewRelixir.Statman.poll/0])
     ]
 
     opts = [strategy: :one_for_one, name: NewRelixir.Supervisor]
-    result = Supervisor.start_link(children, opts)
-
-    :ok = :statman_server.add_subscriber(:statman_aggregator)
-
-    with app_name    <- Application.get_env(:new_relixir, :application_name),
-         license_key <- Application.get_env(:new_relixir, :license_key),
-         true        <- String.valid?(app_name),
-         true        <- String.valid?(license_key) do
-      Application.put_env(:newrelic, :application_name, to_char_list(app_name))
-      Application.put_env(:newrelic, :license_key, to_char_list(license_key))
-
-      {:ok, _} = :newrelic_poller.start_link(&:newrelic_statman.poll/0)
-    end
-
-    result
+    Supervisor.start_link(children, opts)
```

Comment:
We should consider logging a warning message when the app is missing required configs.

Reply:
Done in 69f55aabd68d9a3e4e4cd3d56ecb1c171aa7af0c